### PR TITLE
added intro text to proguard docs

### PR DESCRIPTION
**Pull request by imatwawana**
Adds more intro text to ProGuard page
Fixes typo in page description
Addresses (in part) [issue 4421](https://github.com/getsentry/sentry-docs/issues/4421)

### CODE REVIEW

File: src/platforms/android/common/proguard.mdx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 title: ProGuard
 sidebar_order: 100
-description: "Learn about using ProGuard to upload the ProGuard/R8 mappings, source bundle, and native symbols Sentry."
+description: "Learn about using ProGuard to upload the ProGuard/R8 mappings, source bundle, and native symbols to Sentry."
```
Comment (marandaneto) [inline src/platforms/android/common/proguard.mdx:4]:
```suggestion
description: "Learn about using the Sentry Gradle Plugin or sentry-cli for providing debug symbols to Sentry."
```

what's about this? similar to https://github.com/getsentry/sentry-docs/edit/master/src/platforms/react-native/upload-debug.mdx

File: src/platforms/android/common/proguard.mdx [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 ---
 
-To use ProGuard with Sentry, upload the ProGuard mapping files by either the recommended method of using our Gradle integration or manually by using [sentry-cli](/product/cli/dif/#proguard-mapping-upload).
+If you want to see de-obfuscated logs, you'll need to to use ProGuard with Sentry. To do so, upload the ProGuard mapping files by either the recommended method of using our Gradle integration or manually by using [sentry-cli](/product/cli/dif/#proguard-mapping-upload).
```
Comment (marandaneto) [inline src/platforms/android/common/proguard.mdx:7]:
```suggestion
If you want to see de-obfuscated stack traces, you'll need to use ProGuard with Sentry. To do so, upload the ProGuard mapping files by either the recommended method of using our Gradle integration or manually by using [sentry-cli](/product/cli/dif/#proguard-mapping-upload).
```

File: src/platforms/android/common/proguard.mdx (hunk 1 above)
Comment (marandaneto) [reply src/platforms/android/common/proguard.mdx:4]: is the description shown somewhere? I don't see in the docs page actually.

**Comment by marandaneto** [comment]
@imatwawana the very same intro could fit better the Android wizard, See https://github.com/getsentry/sentry-docs/blob/master/src/wizard/android/index.md?plain=1#L127

File: src/platforms/android/common/proguard.mdx (hunk 1 above)
Comment (imatwawana) [reply src/platforms/android/common/proguard.mdx:4]: It seems like two pages with different content shouldn't have the exact same description, but if you think it's reasonably accurate, then let's go with it. The complaint about the description from the user was just about the typo. What about:
```suggestion
description: "Learn about using the Sentry Gradle Plugin or sentry-cli to upload the ProGuard/R8 mappings, source bundle, and native symbols to Sentry."
```

Comment (imatwawana) [reply src/platforms/android/common/proguard.mdx:4]: > 

No, it's only in the metadata of the page.

Comment (imatwawana) [reply src/platforms/android/common/proguard.mdx:4]: > is the description shown somewhere? I don't see in the docs page actually.

No, it's only in the metadata of the page.

**Pull request merged**